book/src/scalable_lists.md

```diff
@@ -0,0 +1,179 @@
+# Scalable Lists
+
+Sometimes you have more elements than you can display at once.
+The typical solution is to put them into a [`ScrolledWindow`](../docs/gtk/struct.ScrolledWindow.html) so you can scroll through the complete list.
+Howevee, in the most trivial case you would still create one widget per element.
```

Comment:
```suggestion
However, in the most trivial case you would still create one widget per element.
```
not sure how codespell didn't catch this one

Reply:
Good catch.
Codespell is extremely limited.
On the plus side, that keeps the number of false positives to a minimum as well.